### ISSUE TITLE
Should this be included in homeassistant?

### ISSUE THREAD

**Issue by almcelmon** [stjohnjohnson/smartthings-mqtt-bridge#64]
Hey all,

I'm new to the scene, but couldn't this library could be bundled with home assistant via dependencies? I realize it does lots of other things as well, but so does python-nmap and it's included. I feel like the SmartThings/HomeAssist combo is going to continue to get more popular so this would help raise adoption. Since the Raspberry Pi All in One installer sets up mosquito, you should be able to get going right out of the box once you have the SmartThings IDE set up.

**Comment by stjohnjohnson** [comment]
It would be quite nice to be bundled into there.  I would just need to rewrite this in Python (it's really not a lot of code, so it should be straight forward).

**Comment by rabbadab** [comment]
Would be great!

**Comment by AceoStar** [comment]
I'm just getting into all of this, so I'm still not 100% sure how all the HA architecture works, so bare with me a bit while I make some sense of it. :D If this was turned into a python module, I imagine we would somehow set it to auto start if configured, and configure it via the main HA config?

**Comment by vmtyler** [comment]
no, if you built it as a plugin, HA will start it once it's enabled in the config. in this case, HASS already manages MQTT so you could pull the settings from HA. (the ones supplied in the config). basically the stuff you ask for in config.yml would go into the HA config file.
